Dockerfile

```
@@ -31,4 +31,9 @@ WORKDIR /
 COPY --from=builder /workspace/manager .
 USER 65532:65532
 
+# Quay image expiry
+ARG QUAY_IMAGE_EXPIRY
+ENV QUAY_IMAGE_EXPIRY=${QUAY_IMAGE_EXPIRY:-never}
```

Why do we need this ENV?

---

Hmm, I was following the current examples of how this was done. I assumed it was to use the enviroment default value 🤔 

https://github.com/Kuadrant/limitador-operator/blob/1f721dedee02a4768f3d9ace042efbdf5faa60ac/Dockerfile#L19-L25

But this probably can be simpler by doing the following instead if we want to always take it in from the build argument instead
```suggestion
ARG QUAY_IMAGE_EXPIRY=never
```

Let me give it a try and push an update